[2024-04-04T09:48:13.041+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: real_estates.extract_real_estate_urls manual__2024-04-04T08:47:03.043176+00:00 [queued]>
[2024-04-04T09:48:13.060+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: real_estates.extract_real_estate_urls manual__2024-04-04T08:47:03.043176+00:00 [queued]>
[2024-04-04T09:48:13.060+0000] {taskinstance.py:2193} INFO - Starting attempt 8 of 8
[2024-04-04T09:48:13.086+0000] {taskinstance.py:2217} INFO - Executing <Task(_PythonDecoratedOperator): extract_real_estate_urls> on 2024-04-04 08:47:03.043176+00:00
[2024-04-04T09:48:13.100+0000] {standard_task_runner.py:60} INFO - Started process 4930 to run task
[2024-04-04T09:48:13.107+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'real_estates', 'extract_real_estate_urls', 'manual__2024-04-04T08:47:03.043176+00:00', '--job-id', '234', '--raw', '--subdir', 'DAGS_FOLDER/real_estates/__init__.py', '--cfg-path', '/tmp/tmpq7bgdz7v']
[2024-04-04T09:48:13.113+0000] {standard_task_runner.py:88} INFO - Job 234: Subtask extract_real_estate_urls
[2024-04-04T09:48:13.220+0000] {task_command.py:423} INFO - Running <TaskInstance: real_estates.extract_real_estate_urls manual__2024-04-04T08:47:03.043176+00:00 [running]> on host 05a350db99a0
[2024-04-04T09:48:13.682+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='real_estates' AIRFLOW_CTX_TASK_ID='extract_real_estate_urls' AIRFLOW_CTX_EXECUTION_DATE='2024-04-04T08:47:03.043176+00:00' AIRFLOW_CTX_TRY_NUMBER='8' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-04T08:47:03.043176+00:00'
[2024-04-04T09:48:15.584+0000] {urls.py:36} INFO - Found 86115 real estates
[2024-04-04T09:48:15.590+0000] {urls.py:41} INFO - Found 2160 pages
[2024-04-04T09:48:23.571+0000] {local_task_job_runner.py:302} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2024-04-04T09:48:23.575+0000] {process_utils.py:131} INFO - Sending 15 to group 4930. PIDs of all processes in the group: [4930]
[2024-04-04T09:48:23.576+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 4930
[2024-04-04T09:48:23.577+0000] {taskinstance.py:2483} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-04-04T09:48:23.911+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=4930, status='terminated', exitcode=0, started='09:48:12') (4930) terminated with exit code 0
